Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1232,6 +1232,101 @@ def setup_(self, step, stepName, stepDict, k, properties):
     offset = 0.524,
 )
 
+# HCAL-PF Only workflow running HCAL local reco on CPU and PF with Alpaka slimmed for benchmarking
+# - HLT on CPU
+# - HCAL-only reconstruction using Alpaka
+upgradeWFs['PatatrackHCALOnlyAlpakaProfiling'] = PatatrackWorkflow(
+    digi = {
+        # the HLT menu is already set up for using GPUs if available and if the "gpu" modifier is enabled
```

Comment:
can you change this to
```suggestion
        '--procModifiers': 'alpaka', # alpaka modifier activates customiseHLTForAlpaka 
```
in all the Alpaka workflows, to run the HLT alpaka customisation, and update the comments accordingly ?

Reply:
I know that currently `customiseHLTForAlpaka` does not apply the PF-related changes, but @waredjeb is looking into it.

Reply:
Sure, I will update those lines.